app/classifier/index.cjsx

```diff
@@ -25,6 +25,8 @@ auth.listen ->
 
 PROMPT_MINI_COURSE_EVERY = 5
 
+RELOAD_UPP_EVERY = 10
```

Comment:
I'd prefer if this were 5 instead of 10 because I think that Snapshot WI wanted to try 15 as the promotion threshhold, but I understand it's a balancing act wrt API calls

Reply:
Ok, I can make the change.